client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/configure-sms-notification/sms-item-content.tsx

```diff
@@ -0,0 +1,130 @@
+import { Card, Button } from '@automattic/components';
+import { Icon, moreHorizontal } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { useState, useRef } from 'react';
+import Badge from 'calypso/components/badge';
+import PopoverMenu from 'calypso/components/popover-menu';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+
+interface StateMonitorSettingsSMS {
+	number: string;
+	name: string;
+	isDefault: boolean;
```

Comment:
We will not have default phone numbers. It is only for emails.

Reply:
Can we please move this to the types folder? I see this being used at 2 places.

Reply:
Also, I think we will need the following.

```
name: string;
countryCode: string;
phoneNumber: string;
phoneNumberFull: string;
verified: boolean;
```

Since we decided country code and phone number will be 2 different keys, lets keep it like this.